.prettierignore

```diff
@@ -1,4 +1,6 @@
 build
 node_modules
 yarn.lock
-public
\ No newline at end of file
+public
```

Comment:
unnecessary change

Reply:
I only added line 5 and 6 but showing line 4  is also changed

Reply:
I only added line 5 and 6 showing line 4 also changed